src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -206,12 +206,11 @@ void CouplingSchemeConfiguration::xmlTagCallback(
     _config.dtMethod = getTimesteppingMethod(tag.getStringAttributeValue(ATTR_METHOD, VALUE_FIXED));
     if (_config.dtMethod == constants::TimesteppingMethod::FIXED_TIME_WINDOW_SIZE) {
 
-      PRECICE_ASSERT(_minTimeStepSize > 0);
-      PRECICE_CHECK(_config.timeWindowSize >= _minTimeStepSize,
-                    "Time window size has to be larger or equal to the minimal time step used by preCICE. "
+      PRECICE_CHECK(_config.timeWindowSize >= math::NUMERICAL_ZERO_DIFFERENCE,
+                    "Time window size has to be larger or equal to the minimal time step supported by preCICE. "
```

Comment:
We should talk about the numerical zero difference here and we could also give the value.

Reply:
Done: [d631956](https://github.com/precice/precice/pull/1882/commits/d6319565367c0f566901f7212217ddc659b99389). This check was broken anyway :see_no_evil: